### PR TITLE
qutebrowser: update to git master

### PR DESCRIPTION
**Pull request by kirelagin**
@ehahn @jagajaga

### CODE REVIEW

**Comment by jagajaga** [comment]
Tested?

File: pkgs/applications/networking/browsers/qutebrowser/default.nix [hunk 1]
```diff
@@ -1,25 +1,27 @@
-{ stdenv, fetchurl, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2}:
+{ stdenv, fetchgit, python, buildPythonPackage, qt5, pyqt5, jinja2, pygments, pyyaml, pypeg2 }:
 
-let version = "0.2.1"; in
+let version = "0.3-pre"; in
```
Comment (jagajaga) [inline pkgs/applications/networking/browsers/qutebrowser/default.nix]: Better change to the output of `git describe` 
If available. Or change `pre` to the date of commit.

File: pkgs/applications/networking/browsers/qutebrowser/default.nix [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+  };
+
+  # Needs tox
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    python pyyaml pyqt5 jinja2 pygments pypeg2
+  ];
+
+  meta = {
```
Comment (jagajaga) [inline pkgs/applications/networking/browsers/qutebrowser/default.nix]: Do you want to add yourself as a maintainer? If you don't want then add me please.

**Comment by kirelagin** [comment]
Seems to be working for me. Even without crashes (with new Qt, of course).

There is one minor problem:

~~~~
WARNING: QIODevice::write: device not open
  File "/nix/store/fr10smc66xw87c35pk2c5rw7b8jkf8vf-qutebrowser-0.3-pre/lib/python3.4/site-packages/qutebrowser/utils/log.py", line 312, in qt_message_handler
    stack = ''.join(traceback.format_stack())
~~~~

I’m not really sure what’s that about.

Also there is slightly more alarming this:

~~~~
WARNING: defaultServiceProvider::requestService(): no service found for - "org.qt-project.qt.mediaplayer"
  File "/nix/store/fr10smc66xw87c35pk2c5rw7b8jkf8vf-qutebrowser-0.3-pre/bin/.qutebrowser-wrapped", line 10, in <module>
    load_entry_point('qutebrowser==0.2.1', 'gui_scripts', 'qutebrowser')()
  File "/nix/store/fr10smc66xw87c35pk2c5rw7b8jkf8vf-qutebrowser-0.3-pre/lib/python3.4/site-packages/qutebrowser/qutebrowser.py", line 149, in main
    return app.run(args)
  File "/nix/store/fr10smc66xw87c35pk2c5rw7b8jkf8vf-qutebrowser-0.3-pre/lib/python3.4/site-packages/qutebrowser/app.py", line 117, in run
    ret = qt_mainloop()
  File "/nix/store/fr10smc66xw87c35pk2c5rw7b8jkf8vf-qutebrowser-0.3-pre/lib/python3.4/site-packages/qutebrowser/app.py", line 127, in qt_mainloop
    return qApp.exec_()
  File "/nix/store/fr10smc66xw87c35pk2c5rw7b8jkf8vf-qutebrowser-0.3-pre/lib/python3.4/site-packages/qutebrowser/utils/log.py", line 312, in qt_message_handler
    stack = ''.join(traceback.format_stack())
~~~~

This has something to do with interoperation between Qt and gstreamer (by the way I think Qt in nixpkgs is build against legacy gstreamer-0.10, which is not particularly cool).

I believe neither is related to this update.

File: pkgs/applications/networking/browsers/qutebrowser/default.nix (hunk 1 above)
Comment (kirelagin) [inline pkgs/applications/networking/browsers/qutebrowser/default.nix]: Well, the point of `0.3-pre` is that I believe that’s _actually_ the commit that will become 0.3. But, fine, I’ll change to the date.

Comment (jagajaga) [inline pkgs/applications/networking/browsers/qutebrowser/default.nix]: Oh! Sorry, I forgot that nix is ok with `pre` as a part of a version.

**Comment by kirelagin** [comment]
> jagajaga merged commit e44d325 into NixOS:master 2 minutes ago

Oh come on, I just pushed a new squashed commit with a different version string.
Anyway, it’s too late now, let it be `0.3-pre`. :sleeping:

**Comment by jagajaga** [comment]
@kirelagin everything was ok.

**Comment by The-Compiler** [comment]
v0.3 is released in the meantime - the only difference to the commit listed here is the [version number change](https://github.com/The-Compiler/qutebrowser/commit/b3cd31a808789932a0a4cb7aa8d9280b6d3a12e7).

If you aren't yet, I suggest subscribing to the [mailinglist](https://lists.schokokeks.org/mailman/listinfo.cgi/qutebrowser) so you know about new releases - I also post weekly update mails there, so if that's too annoying, let me know and I'll start a separate `-annouce` list.

Also if someone could open a PR to update [INSTALL.asciidoc](https://github.com/The-Compiler/qutebrowser/blob/master/INSTALL.asciidoc) for NixOS that'd be very much appreciated!

**Pull request merged**